examples/qwik/github-oauth/src/auth/lucia.ts

```diff
@@ -0,0 +1,35 @@
+import fs from "node:fs";
+import { betterSqlite3 } from "@lucia-auth/adapter-sqlite";
+import { github } from "@lucia-auth/oauth/providers";
+import sqlite from "better-sqlite3";
+import { lucia } from "lucia";
+import { qwik } from "lucia/middleware";
+// import "lucia/polyfill/node";
+
+const db = sqlite(":memory:");
+db.exec(fs.readFileSync("schema.sql", "utf8"));
+
+export const auth = lucia({
+	adapter: betterSqlite3(db, {
+		user: "user",
+		session: "user_session",
+		key: "user_key"
+	}),
+	env: process.env.NODE_ENV === "development" ? "DEV" : "PROD",
+	middleware: qwik(),
+	sessionCookie: {
+		expires: false
+	},
+	getUserAttributes: (data) => {
+		return {
+			githubUsername: data.username
```

Comment:
Can you rename `githubUsername` to just `username`? I changed it like yesterday 😅

Reply:
I changed it when I saw that, but missed this one.